content/general/concepts/tuple/tuple.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Tuple'
+Description: 'A tuple is a data structure consisting of an ordered collection with an arbitrary number of elements.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Tuples'
+  - 'Data Structures'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In computer science, a tuple is a data structure consisting of an ordered collection with an arbitrary number of elements. A tuple of "N" elements, where "N" is a positive integer, is generally referred to as an N-tuple. Many computer languages implement a tuple as its own structure or object type.
+
+Typically, a tuple implementation in any programming language has the following characteristics:
+
+- It can contain any object or [data type](https://www.codecademy.com/resources/docs/general/data-types) in any combination.
+- It allows duplicate entries.
+- It has a specified order.
```

Comment:
Is this referring to how tuple items are organized? (e.g. order by insertion)

Reply:
Yes.

Reply:
Did we want to be explicit here on its specified order by including that it's ordered by insertion?

Reply:
I expanded on the point, though I didn't use the phrase "order of insertion" since "insertion" implies mutability, and tuples are generally created all at once.